crates/typst-library/src/foundations/symbol.rs

```diff
@@ -192,7 +210,18 @@ impl Symbol {
             if list.iter().any(|(prev, _)| &v.0 == prev) {
                 bail!(span, "duplicate variant");
             }
-            list.push((v.0, SymChar::pure(v.1)));
+            if !v.0.is_empty() {
+                for modifier in v.0.split('.') {
+                    if !is_ident(modifier) {
+                        bail!(
+                            span,
+                            "invalid symbol modifier: {}",
+                            crate::foundations::Repr::repr(modifier),
```

Comment:
```suggestion
                            modifier.repr(),
```

Reply:
This does not work because it requires importing the `Repr` trait, and prevents us from using `Repr` as the name of the type contained in `Symbol`.